EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationTrack.h

```diff
@@ -53,21 +55,22 @@ class VisualisationTrack
   /// Simplifies passing parameters to constructor of VisualisationTrack
   /// by providing their names
   struct VisualisationTrackVO {
+    float time;
     int charge;
-    double energy;
-    int ID;
+    //  double energy;
+    //  int ID;
     int PID;
-    double mass;
-    double signedPT;
+    //  double mass;
+    //  double signedPT;
     double startXYZ[3];
```

Comment:
same here, float would be fully enough for all these data members.

Reply:
done